### ISSUE TITLE
manual self-verification of a given device only gives emoji option, not QR

### ISSUE THREAD

**Issue by ara4n** [vector-im/riotX-android#1245]
if i try to verify Riot/Web from Riot/X's session management in settings, it only lets me try to verify it by emoji:

**Comment by BillCarsonFr** [comment]
You current session does not trust the MSK.
Currently in this case the verify will launch a legacy start SAS

**Comment by BillCarsonFr** [comment]
We probably have an issue here. 
The manage session will display local trust:
![image](https://user-images.githubusercontent.com/9841565/79547207-de849480-8093-11ea-848c-c56f80c08f10.png)

It should propably not do that.. The current device should be red shield and the others blacks shields?
And when you try to verify another session it should ask you first to complete security?

@nadonomy  what do you think

**Comment by BillCarsonFr** [comment]
The state of user profile is also probably a bit off
<img width="390" alt="image" src="https://user-images.githubusercontent.com/9841565/79551030-6c16b300-8099-11ea-956d-6d63f8eeb31a.png">

**Comment by nadonomy** [comment]
> @nadonomy what do you think

Yeah I think this makes sense. So effectively if a user is using cross-signing, they sign into a new device and then skip verifying it on login and go to settings, we echo locally how other users see their pool of devices?

So this device would be untrusted (red) and once cross-signed would turn green, and the other devices would update from black to represent either their trusted (green) or untrusted (red) state?

**Comment by nadonomy** [comment]
@BillCarsonFr & I had a call on this to discuss in more detail, and we think it makes sense to:

In Settings
- Decorate your local session as red (as it's untrusted)
- Decorate the other other sessions as black
- Then, when tapping on your local session, we trigger bootstrapping
- And, when tapping on an unknown session, we have an extra [Complete Security] action to trigger bootstrapping

When viewing your own profile
- Decorate the same, where black devices are labelled as 'Unknown'
